util/features/features.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
These should be moved under `third_party/kubernetes-features`

Reply:
Also let's add a README.md like the one in https://github.com/kubernetes-sigs/cluster-api/blob/master/third_party/kubernetes-drain/README.md

Reply:
possibly once kubeadm moves into a module, the remote package can be used instead of the local fork. adding a "TODO" note also make sense, as some contributors search for that.